feat: save labeled data to Azure blob

ferreroxc opened the pull request:
- append sasToken to URI
- read [account, container name and sas token] from labeler_settings
- create a blob
- write data to blob
- prepend name of output file with "labeled_date_time_"
- run npm update

Commit:
feat: save labeled data to Azure blob

- append sasToken to URI
- read [account, container name and sas token] from labeler_settings
- create a blob
- write data to blob
- prepend name of output file with "labeled_date_time_"
- run npm update

ginic commented on src/modules/labeler.js:
```diff
@@ -648,8 +602,29 @@ export class LabelerApp {
+				const containerClient = blobServiceClient.getContainerClient(azureContainerName);
+			  
+				const currentTime = new Date();
+				const blobName = "labeled_" + currentTime.toLocaleDateString('de-DE') + "_" + currentTime.toLocaleTimeString('de-DE') + "_"+ fileName;
```
Any particular reason you chose to use the German 'de' locale? It adds `.` to the middle of the filename, which isn't preferred (it's better to let that be only used for the file type suffix). I would recommend using a format like `YYYY-MM-DD-HHMMSS`, or something that doesn't contain special characters like dots and colons.

ferreroxc replied:
Because "de" is 24-hour and does not contain "/", which makes directory. Let me see how to change this format to `YYYY-MM-DD-HHMMSS`

ferreroxc replied:
Changed to "labeled_2023-05-31-104538_test_0.geojson" in this [commit](https://github.com/UMassCDS/satellite-imagery-labeling-tool/pull/8/commits/e71080cd0b05cc3ca6913467989b1c68ab45de05)!

tnmygrwl replied:
thank @ferreroxc and @ginic. Looks good to me now. Merging now. 👍🏾

Commit:
feat: pass SAS in env var to docker

- add instruction on how to add env var in doc
- changed SAS from hardcoding in labeler_settings.js to passing env var

Commit:
feat: use YYYY-MM-DD-HHMMSS format in saved filename

Commit:
add changelog
